theforeman.org/scripts/debian/execute_pbuilder_dependency.sh

```diff
@@ -7,15 +7,28 @@ cd dependencies/${os}
 mkdir build-${project}
 cd build-${project}
 
-# Import variables from the project, allowing it to override gem2deb behaviour
+BUILD_TYPE=gem
+# Import variables from the project, allowing it to override behaviour
 if [ -e ../${project}/build_vars.sh ]; then
   . ../${project}/build_vars.sh
 fi
 
-# Figure out package version and gem2deb it
+# Figure out package version and get it
 VERSION=$(head -n1 ../${project}/changelog|awk '{print $2}'|sed 's/(//;s/)//'|cut -f1 -d-|cut -d: -f2)
-gem fetch ${project} -v "=${VERSION}"
-../../gem2deb ${project}-${VERSION}.gem --debian-subdir ../${project} --only-source-dir
+if [ "${BUILD_TYPE}" == "python" ]; then
```

Comment:
I think `==` is a bashism. If you do that, isn't this even better?
```suggestion
if [[ $BUILD_TYPE == "python" ]]; then
```

Reply:
Gah, yes it is. And given the script is bash anyways, [[ is faster, being a built-in